clients/roscpp/include/ros/node_handle.h

```diff
@@ -112,11 +112,17 @@ namespace ros
     /**
      * \brief Parent constructor
      *
-     * This version of the constructor takes a "parent" NodeHandle, and is equivalent to:
+     * This version of the constructor takes a "parent" NodeHandle.
+     * If the passes ns is relative (does not start with a slash), it is equivalent to calling:
```

Comment:
"passed" not "passes" and should "ns" be expanded to "namespace" since "ns" can also mean "nanoseconds"?